pom.xml

```diff
@@ -70,6 +70,37 @@
                     <autoReleaseAfterClose>false</autoReleaseAfterClose>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>com.coveo</groupId>
+                <artifactId>fmt-maven-plugin</artifactId>
+                <version>2.10</version>
```

Comment:
Version 2.10 requires Java 11 at least.
If we want to build with Java 8, we need to downgrade to 2.9.

Reply:
Java 11 is not currently an option

Reply:
Probably why the build is failing

Reply:
OK, I downgraded to 2.9. I hope this won't introduce incompatibilities between the versions of google-formatter used by this plugin and IDE plugins.